Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -1675,43 +1800,51 @@ NativeArray<byte> outputStream
             return node;
         }
         
-        int AddMesh(UnityEngine.Mesh uMesh) {
-            int meshId;
-            
+        void AddMeshAndSkin(UnityEngine.Mesh uMesh, Transform[] bones, out int meshId, out int skinId)

```

Comment:
Idea: Consider returning a bool value, true in case of success, false otherwise.

Reply:
Why: more explicit